tough-kms/Cargo.toml

```diff
@@ -16,16 +16,16 @@ aws-sdk-rust-rustls = ["aws-config/rustls", "aws-sdk-kms/rustls"]
 [dependencies]
 tough = { version = "0.16", path = "../tough", features = ["http"] }
 ring = { version = "0.17", features = ["std"] }
-aws-sdk-kms = "0.36"
-aws-config = "0.57"
+aws-sdk-kms = "1"
+aws-config = "1"
```

Comment:
Is this safe to do as there has been pretty big variability with 0.x.0 versions in the past?

Reply:
Yes, we need to make the jump to 1.x in all our projects so that we don't have multiple versions in our builds. Especially when EC2 is involved because it is an enormous compile time.